File: apzkr-pzpi-21-10-mykolchuk-mykhailo/Task1-Server/Domain/Models/Car.cs
namespace APZ_backend.Models
{
    public class Car
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public int OwnerId { get; set; }
        public User Owner { get; set; }
        public string Type { get; set; }
        public DateTime Added { get; set; }
        public int? SensorId { get; set; }
        public Sensor? Sensor { get; set; }
        public ICollection<Subordinate>? Drivers { get; set; }
        public ICollection<Record>? Records { get; set; }
    }
}

File: apzkr-pzpi-21-10-mykolchuk-mykhailo/Task4-MobileClient/app/src/main/java/com/example/apz_mobile/models/Car.kt
package com.example.apz_mobile.models

import android.os.Parcelable
import kotlinx.parcelize.Parcelize

@Parcelize
data class Car(
    val id: Int,
    val name: String,
    val ownerId: Int,
    val type: String,
    val added: String,
    val sensor: SensorForCar?,
    val drivers: List<Driver> = listOf()
) : Parcelable
{
    override fun toString(): String {
        return "$name ($type)"
    }
}

@Parcelize
data class SensorForCar(val id: Int, val name: String) : Parcelable

@Parcelize
data class Driver(val id: Int, val name: String, val email: String, val regDate: String) :
    Parcelable